src/huggingface_hub/keras_mixin.py

```diff
@@ -123,15 +85,11 @@ def _create_model_card(
     model_card += "\n## Intended uses & limitations\n\nMore information needed\n"
     model_card += "\n## Training and evaluation data\n\nMore information needed\n"
     if hyperparameters is not None:
-        model_card += "\n## Training procedure\n"
```

Comment:
Why is this removed? I don't know if it needs to be there or not, but just curious as to why its gone now.

Reply:
I thought it was redundant. I can bring it back.